src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -882,7 +902,7 @@ def get_request_status_logs(
 ) -> AbstractPage[ExecutionLog]:
     """Returns all the execution logs associated with a given privacy request ordered by updated asc."""
 
-    get_privacy_request_or_error(db, privacy_request_id)
+    get_privacy_request_or_error(db, privacy_request_id, error_if_deleted=False)
```

Comment:
I allow `get` operations on the deleted privacy request , but not update ones. TBH there weren't any specific requirements regarding this on the ticket, so I used my best judgement. Any feedback / opinions are welcome

Reply:
This is a great thing to run by Michael early on!   There's a case to be made either way.

Maybe I would strive for consistency here:`get_request_status_logs`, `view_uploaded_manual_webhook_data`, `view_uploaded_erasure_manual_webhook_data`, and `get_individual_privacy_request_tasks` return very specific details about potentially deleted privacy requests by default. 

But the search endpoint which is the place to get primary privacy request details does _not_ return deleted privacy requests by default.  This latter detail tips me towards wanting to hide everything by default or add query params to the GET ones so the default is "don't show" instead of "show"


![Screenshot 2024-09-24 at 3 05 00 PM](https://github.com/user-attachments/assets/0cda3315-c2d5-4144-a851-e23d0dc66b19)

Reply:
Confirmed the requirements here https://ethyca.atlassian.net/browse/PROD-2700?focusedCommentId=44123